templates/pages/person.html

```diff
@@ -10,18 +11,18 @@
 
     <div class="row">
       <div class="columns medium-8">
-        
+
         {{> records/record-description}}
       </div>
       <div class="columns medium-4">
 
-        <div class="panel">
+        <!-- <div class="panel">
```

Comment:
It's commented out [for the demo tomorrow](https://github.com/TheScienceMuseum/collectionsonline/issues/106#issuecomment-229942865). It may or may not be back after that